src/components/FileReader.vue

```diff
@@ -8,11 +8,18 @@ const importFile: any = useImportStore()
 
 const file: any = reactive({})
 const content: any = ref("")
-const loading: any = ref(false)
+const loading = ref(false)
 
-function readMatlabTxtFile(): void {
-    file.value = docFile.value.files[0];
+function readGeneralFileInput(): void {
+    if (docFile.value.files.length > 0) {
```

Comment:
What if a user uploads multiple files?

Reply:
Previous parsed file removed, file parser again parses as normal. We can make array if we want more imports at some point but outside of scope.